components/chart/index.tsx

```diff
@@ -31,7 +32,7 @@ function Chart({
     isValid: true,
     message: '',
   });
-
+  const [selectedNodeID, setSelectedNodeID] = useState<number | null>(null);
```

Comment:
I think no need to have a new state variable, we can just use setSelectedModule

Reply:
done